### PR TITLE
Document usage of review by page scripts in Windows Terminal

### PR DESCRIPTION
**Pull request by codeofdusk**
### Link to issue number:
Follow-up of #14021.

### Summary of the issue:
New "review by page" scripts were added to NVDA, but their utility in Windows Terminal is not immediately apparent.

### Description of how this pull request fixes the issue:
Update documentation and what's new.

### Testing strategy:
N/A (no code changes).

### Known issues with pull request:
I imagine that these commands will also work in an upcoming (23xx) version of Windows Console (`conhost`), but with the switch to `wt` by default in Windows, this probably isn't worth noting.

### Change log entries:
See PR

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English

### CODE REVIEW

**Comment by codeofdusk** [comment]
I'll mark PR ready for review once Windows Terminal changes land.

**Comment by codeofdusk** [comment]
This PR should be ready for 2022.4 by the end of the week, once I know which wt version will have the feature.

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -1280,6 +1280,9 @@ The following built-in Windows Console keyboard shortcuts may be useful when [re
 | Scroll down | control+downArrow | Scrolls the console window down, so later text can be read. |
 | Scroll to start | control+home | Scrolls the console window to the beginning of the buffer. |
 | Scroll to end | control+end | Scrolls the console window to the end of the buffer. |
+
+When using Windows Terminal version 1.18 and later, you may use NVDA's Move to previous page in review/Move to next page in review commands to scroll visible screens of output.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:1284]: Can you add a link to the relevant section of the user guide, and change the wording to improve readability?

```suggestion
When using Windows Terminal version 1.18 and later, the gestures ["Move to previous page in review" and "Move to next page in review" commands <add link to section of user guides>] can be used to scroll visible screens of output.

```

**Comment by seanbudd** [comment]
Has 1.18 landed with this change in it?

**Comment by codeofdusk** [comment]
Why was this PR closed?

> Has 1.18 landed with this change in it?

It hasn't yet landed, but 1.18 will include the terminal-side changes.

**Comment by seanbudd** [comment]
>It hasn't yet landed, but 1.18 will include the terminal-side changes.

Is there a public reference for this?
We tend not to document unreleased features.

>Why was this PR closed?
This was auto closed because the target branch was merged (which is weird, normally it would re-target beta).

**Comment by seanbudd** [comment]
it's weirdly not letting me re-open or re-target this PR. When will 1.18 land? I don't think this will make 2022.4 unless 1.18 will land in the next month.

**Comment by codeofdusk** [comment]
Once a public ref lands, might that help you?

**Comment by seanbudd** [comment]
Yes that would help

File: user_docs/en/userGuide.t2t (hunk 1 above)
Comment (seanbudd) [reply user_docs/en/userGuide.t2t:1284]: this has not been addressed

File: user_docs/en/changes.t2t [hunk 2]
```diff
@@ -12,6 +12,7 @@ What's New in NVDA
   - Laptop: ``NVDA+ctrl+shift+.``.
   -
 - Introduced new commands to move the review cursor by page where supported by the application. (#14021)
+  - These are especially useful for moving across screens of text, such as paginated output, in Windows Terminal version 1.18 and later.
```
Comment (seanbudd) [inline user_docs/en/changes.t2t:15]: I think In-Process may be a better way to communicate this.
As this feature has not been introduced on the terminal side yet, and the 2022.4 translation freeze is over, this PR can no longer make 2022.4

**Comment by seanbudd** [comment]
Closing this PR. The work appears to be abandoned by MS

**Pull request closed without merging**